### PR TITLE
addingMetricsSupportInKafkaConnector

### PR DESCRIPTION
**Pull request by xinlian12**
Related to https://github.com/Azure/azure-sdk-for-java/issues/38765

### Design
azure-cosmos has [metrics support](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos/docs/Metrics.md), and exposed multiple API so that customer can config with metrics category, tags, percentiles etc.

In kafka connector, instead of exposing those capabilities through configs, we allow customers to configure through environment variable/system properties + enable metrics collection by using java agents. 

**Example with use AppInsights java agent**
`export KAFKA_OPTS="-javaagent:$base_dir/applicationinsights-agent-3.5.1.jar -DCOSMOS.METRICS_CONFIG={\"metricCategories\":\"[OperationDetails]\"}"`

```
 System.setProperty(
            "COSMOS.METRICS_CONFIG",
           "{\"metricCategories\":\"[OperationSummary, RequestSummary]\","
               + "\"tagNames\":\"[Container, Operation]\","
               + "\"sampleRate\":0.5,"
               + "\"percentiles\":[0.90,0.99],"
               + "\"enableHistograms\":false,"
               + "\"applyDiagnosticThresholdsForTransportLevelMeters\":true}");
```
image

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java [hunk 1]
```diff
@@ -175,6 +174,19 @@ public class Configs {
...
+    //            System.setProperty(
+    //                "COSMOS.METRICS_CONFIG",
+    //                "{\"metricCategories\":\"[OperationSummary, RequestSummary]\","
+    //                + "\"tagNames\":\"[Container, Operation]\","
+    //                + "\"sampleRate\":0.5,"
+    //                + "\"percentiles\":[0.90,0.99],"
+    //                + "\"enableHistograms\":false,"
+    //                + "\"applyDiagnosticThresholdsForTransportLevelMeters\":true}");
+    public static final String METRICS_CONFIG = "COSMOS.METRICS_CONFIG";
+    public static final String DEFAULT_METRICS_CONFIG = new CosmosMicrometerMetricsConfig().toJson();
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java:188]: If its a new instance of `CosmosMicrometerMetricsConfig()` class with default variables and their declared values converted to a string, why do we need the instance at first place? Can't we use something static string to represent the same new instance as a string without creating extra memory for the instance?

**Review by kushagraThapar — APPROVED** [review]
Overall looks good to me, thanks @xinlian12

**Pull request merged**